config.js

```diff
@@ -571,7 +571,12 @@ var config = {
 
     // Allow all above example options to include a trailing comma and
     // prevent fear when commenting out the last value.
-    makeJsonParserHappy: 'even if last key had a trailing comma'
+    makeJsonParserHappy: 'even if last key had a trailing comma',
+
+    // The URL of the moderated rooms microservice, if available. If it
+    // is present, a link to the service will be rendered on the welcome page,
+    // otherwise the app doesn't render it.
+    moderatedRoomServiceUrl: 'https://moderated.jitsi.net'
```

Comment:
Please don't put this after the end marker, `makeJsonParserHappy` should be the very last option here.

Also, don't leave it un-commented. The code should handle undefined.